src/query/inc/qSqlparser.h

```diff
@@ -96,9 +96,11 @@ typedef struct SIntervalVal {
   SStrToken          offset;
 } SIntervalVal;
 
+typedef struct tSqlExpr tSqlExprTimestamp;
+
 typedef struct SRangeVal {
-  void          *start;
```

Comment:
SRangeVal is actually a range of timestamps in tSqlExpr. Use typedef alias to indicate its intention to improve code readability.